.github/workflows/test.yml

```diff
@@ -2,10 +2,12 @@
 name: test
```

Comment:
Yeah, they could be different jobs, sure. I never bothered to change this since they were already separated from the beginning. I guess it's for semantics purposes 🤷‍♂️.

As I see it (but this is me only, I guess) `ci` will do the "unit tests". `test` will do something else. (and before these changes one of the two could actually be launched by pressing a button on the UI - it's those `_dispatch` options).

I can join them. @starbelly, do you feel it's better if these go to the same file now that they're dispatched in the same conditions?